adambushman/public-library-web-app
# Account summary page build

## Description

When a user is authenticated, they have access to an "account" page. This is where they see all of their own stuff, including history of checkouts and any fees accrued to the account. This is different than the high-level view a staff member could see for any user account.

I built out this page to pull such data and display it nicely. It references the stored user email in the `$_SESSION`. I also added the necessary data in `LIB_CHECKOUT`, `LIB_CHECKOUT_ITEM`, and `LIB_FEES`. 

## How to test

Launch the website, sign-in to a user account, and click "Account" in the nav bar. Explore what there is to see there. In theory, a user should also be able to edit their own details from this section. Would likely need to be tied to the other work done by the group.
